torch/_dynamo/variables/functions.py

```diff
@@ -1280,13 +1290,14 @@ def call_function(
                 ],
             )
         elif isinstance(self.value, types.WrapperDescriptorType):
-            msg = (
-                f"Graph break due to unsupported wrapper descriptor {self.value}. "
-                f"Please file an issue on GitHub "
-                f"so the PyTorch team can add support for it. "
+            unimplemented_v2(
```

Comment:
I don't think this is an unconditional graph break on WrapperDescriptorTypes - this is a graph break when a skipped function happens to be a WrapperDescriptorType. So maybe this message should be more similar to the normal skipped function graph break message below.

Reply:
Can't seem to figure out a repro that hits this path. Thinking more it's a bit unclear why we need to special case on wrapper descriptor here. I'll try removing this and see what CI says.